Update base styles

andyfu84 opened the pull request.

andyfu84 committed:
updated side menu

vicary reviewed (changes requested):
Minified assets should be excluded from source code.

andyfu84 committed:
change minified files to src files

andyfu84 committed:
uploaded changed files to .private/src

andyfu84 committed:
updated user controller js

vicary committed:
Update main.less

1. Use `overwrite.less` instead of `overwrite.min.css`
2. Move import to the end of file to ensure overwrites
3. Add `position: relative;` to confine route templates inside main section.

vicary committed:
Update overwrite.less

1. Tidy up css into less syntax
2. Change overly aggressive class selectors to ids

vicary committed:
Update index.html

Tidy up.

vicary committed:
Update overwrite.less

Add support for glyphicon brand icons, should be dropped when we migrate to bootstrap 4.

vicary committed:
Update generic-model-list-view.tpl

Use full width to match the new design.
